lib/ret_web/channels/hub_channel.ex

```diff
@@ -140,12 +140,32 @@ defmodule RetWeb.HubChannel do
     {:noreply, socket}
   end
 
-  def handle_in("naf" = event, %{"data" => %{"isFirstSync" => true}} = payload, socket) do
-    data =
-      payload["data"] |> Map.put("creator", socket.assigns.session_id) |> Map.put("owner", socket.assigns.session_id)
+  def handle_in("naf" = event, %{"data" => %{"isFirstSync" => true, "template" => naf_template}} = payload, socket) do
```

Comment:
i'd add a comment noting that this match captures all inbound NAF messages that result in spawned objects